### PR TITLE
Fix contentItem size not change after set renderwindow's width/height

### PR DESCRIPTION
**Pull request by wineee**
This will cause topbar not visable when only one output

### CODE REVIEW

File: src/server/qtquick/woutputrenderwindow.h [hunk 1]
```diff
@@ -24,6 +24,8 @@ class WAYLIB_SERVER_EXPORT WOutputRenderWindow : public QQuickWindow, public QQm
     Q_OBJECT
     Q_DECLARE_PRIVATE(WOutputRenderWindow)
     Q_PROPERTY(WWaylandCompositor *compositor READ compositor WRITE setCompositor REQUIRED)
+    Q_PROPERTY(int width READ width WRITE setWidth NOTIFY widthChanged)
```
Comment (zccrs) [inline src/server/qtquick/woutputrenderwindow.h:27]: qreal

File: src/server/qtquick/woutputrenderwindow.h [hunk 2]
```diff
@@ -49,6 +51,12 @@ public Q_SLOTS:
     void render();
     void scheduleRender();
     void update();
+    void setWidth(int arg);
+    void setHeight(int arg);
+
+Q_SIGNALS:
+    void widthChanged(int arg);
```
Comment (zccrs) [inline src/server/qtquick/woutputrenderwindow.h:58]: Don't add new signal, use QWindow::widthChanged

**Review by zccrs — CHANGES_REQUESTED** [review]

File: src/server/qtquick/woutputrenderwindow.h (hunk 2 above)
Comment (wineee) [reply src/server/qtquick/woutputrenderwindow.h:58]:
```
In file included from /home/rewine/dev/waylib/src/server/qtquick/woutputrenderwindow.cpp:1297:
/home/rewine/dev/waylib/build/src/server/waylibserver_autogen/include/moc_woutputrenderwindow.cpp: 在函数‘void checkNotifySignalValidity_CLASSWaylibSCOPEServerSCOPEWOutputRenderWindowENDCLASS(Waylib::Server::WOutputRenderWindow*)’中:
/home/rewine/dev/waylib/build/src/server/waylibserver_autogen/include/moc_woutputrenderwindow.cpp:271:20: 错误：no matching function for call to ‘Waylib::Server::WOutputRenderWindow::widthChanged()’
  271 |     t->widthChanged();
      |     ~~~~~~~~~~~~~~~^~

```

File: src/server/qtquick/woutputrenderwindow.cpp [hunk 3]
```diff
@@ -1240,6 +1240,30 @@ void WOutputRenderWindow::update()
...
+qreal WOutputRenderWindow::height()
+{
+    return contentItem()->height();
+}
+
+void WOutputRenderWindow::setWidth(qreal arg)
+{
+    QQuickWindow::setWidth(arg);
+    contentItem()->setWidth(arg);
+    Q_EMIT widthChanged(arg);
```
Comment (zccrs) [inline src/server/qtquick/woutputrenderwindow.cpp:1257]: Don't emit signal here, connect to contentItem->widthChanged

**Review by zccrs — CHANGES_REQUESTED** [review]

**Review by zccrs — APPROVED** [review]

**Pull request merged**